server/src/main/scala/org/labrad/browser/VaultApi.scala

```diff
@@ -106,6 +107,10 @@ class VaultApi(cxn: LabradConnection, client: VaultClientApi)(implicit ec: Execu
 
   import VaultApi._
 
+  private val DATA_AVAILABLE = 4444
```

Comment:
These are message IDs that we are free to chose arbitrarily; we tell the server to send each message to a particular ID and then use that to know how to handle the messages we get. I will add some comments to make this a bit more clear.